### PR TITLE
fix: Adjust My Collection placeholder

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: **Bugfix**

### Description

This PR adjusts the My Collection placeholder and removes the initial spinner.

| | |
| --- | --- |
| Simulator Screen Shot - iPhone 12 mini - 2022-01-31 at 16 47 43 | Simulator Screen Shot - iPhone 12 mini - 2022-01-31 at 16 48 58 |

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Adjust My Collection placeholder - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Adjust My Collection placeholder - ole)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 72f1bffeca090d2389d1c180a3480223ea873b7a
</p>

File: src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx [hunk 1]
```diff
@@ -224,7 +224,11 @@ export const MyCollectionAndSavedWorksQueryRenderer: React.FC<{}> = ({}) => (
     <QueryRenderer<MyCollectionAndSavedWorksQuery>
       environment={defaultEnvironment}
       query={MyCollectionAndSavedWorksScreenQuery}
-      render={renderWithLoadProgress(MyCollectionAndSavedWorksFragmentContainer)}
+      render={renderWithPlaceholder({
+        Container: MyCollectionAndSavedWorksFragmentContainer,
+        renderPlaceholder: () => <MyCollectionPlaceholder />,
+        renderFallback: () => null,
```
Comment (MrSltun) [inline src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx:230]: **question**: When does `renderFallback` gets called?

**Review by MrSltun — APPROVED** [review]
Looks good to me, amazing work Ole 🙏

File: src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx (hunk 1 above)
Comment (olerichter00) [reply src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx:230]: Actually, we shouldn't override it here. I've added a fix here: https://github.com/artsy/eigen/pull/6134

**Pull request merged**